sample-data.n3

```diff
@@ -897,50 +979,57 @@
 <http://vivo.mydomain.edu/individual/n700>
         a                      vivo:Committee ;
         rdfs:label             "University Undergraduate Curriculum Committee"@en-US ;
+        rdfs:label             "Commité universitaire pour le cursus de baccaleauréat"@fr-CA ;
         vivo:contributingRole  <http://vivo.mydomain.edu/individual/n7238> .
 
 <http://vivo.mydomain.edu/individual/n3854>
         a                      vivo:Committee ;
         rdfs:label             "College of Arts and Humanities Tenure Committee"@en-US ;
+        rdfs:label             "Commité des mandats du College des Arts et Humanités"@fr-CA ;
         vivo:contributingRole  <http://vivo.mydomain.edu/individual/n1630> .
 
 <http://vivo.mydomain.edu/individual/n1158>
         a                vivo:Postdoc ;
         rdfs:label       "Martinez, Claudia"@en-US ;
+        rdfs:label       "Martinez, Claudia"@fr-CA ;
         obo:ARG_2000028  <http://vivo.mydomain.edu/individual/n4712> ;
         obo:RO_0000053   <http://vivo.mydomain.edu/individual/n2816> , <http://vivo.mydomain.edu/individual/n1515> ;
         vivo:relatedBy   <http://vivo.mydomain.edu/individual/n2543> , <http://vivo.mydomain.edu/individual/n6053> , <http://vivo.mydomain.edu/individual/n5058> .
 
 <http://vivo.mydomain.edu/individual/n6748>
         a                         bibo:Journal ;
         rdfs:label                "Journal of Political Rhetoric"@en-US ;
+        rdfs:label                "Journal de Politique Rhétorique"@fr-CA ;
```

Comment:
It's disputable if journal titles should be translated.

Reply:
That's a good point @hauschke ! 
From a technical perspective however, it does not make any difference if the string tagged as french is actually written in english or another language.